plugin/src/test/kotlin/org/openrewrite/gradle/RewriteRunTest.kt

```diff
@@ -573,7 +635,7 @@ class RewriteRunTest : RewritePluginTest {
                 }
                 
                 dependencies {
-                    implementation("com.fasterxml.jackson.core:jackson-core:2.16.1")
+                    implementation("com.fasterxml.jackson.core:jackson-core:2.16.0")
```

Comment:
Why this change?
```suggestion
                    implementation("com.fasterxml.jackson.core:jackson-core:2.16.1")
```

Reply:
@timtebeek I believe the story is like this:
* the original test was written when the latest version of `jackson-core` was `2.16.0`. Back then it was passing only accidentally, because there was a typo in YAML recipe config (`nevVersion` instead of `newVersion`) and `UpgradeDependencyVersion` uses default of `latest.release` when the `newVersion` is not provided
* newer version of `jackson-core` was released (`2.16.1`) which caused the test to fail
* I noticed the test failing locally and I fixed the typo `nevVersion` -> `newVersion`
* on master it was "fixed" by changing `2.16.0` to `2.16.1` in the assertion
* I rebased on master and changed it back to `2.16.0` because it's the correct value expected by this test 😉

Reply:
Thanks for the explanation! Makes sense now indeed, and thanks for the fix